es_ES/ui-performance/index.md

```diff
@@ -0,0 +1,404 @@
+
+En Android Studio e IntelliJ IDEA:
+
+1. Selecciona **View > Tool Windows > Flutter Inspector**.
+1. En la barra de herramientas, selecciona el icono que parece una biblioteca (<img src="images/performance-overlay-icon.png" alt="icon that resembles a bookshelf">).
+
+![IntelliJ Flutter Inspector Window](/images/intellij/visual-debugging.png)<br>
+
+El Flutter Inspector está disponible en Android Studio e IntelliJ.
+Obtenga más información sobre lo que el Inspector puede hacer en el documento
+[Inspecciona tu UI](/inspector/), así como el archivo
+[Flutter Inspector talk](https://www.youtube.com/watch?v=JIcmJNT9DNI)
+presentado en el DartConf 2018.
+
+#### En VS Code
+
+1. Seleciona **View > Command Palette…** para mostrar la paleta de comandos.
+1. En el campo de texto, escribes "performance" y selecciona
+   **Toggle Performance Overlay** de la lista que aparece.
+   Si este comando no está disponible, asegúrate de que la aplicación esté ejecutándose.
+
+#### Desde la línea de Comando
+
+Conmute la ventana de rendimiento con la tecla **P** desde la línea de comando.
+
+#### Programáticamente
+
+Puede habilitar programáticamente el widget PerformanceOverlay estableciendo la propiedad `showPerformanceOverlay` en `true` en el constructor de MaterialApp o WidgetsApp:
+
+<!-- skip -->
+{% prettify dart %}
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      [[highlight]]showPerformanceOverlay: true,[[/highlight]]
+      title: 'My Awesome App',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: MyHomePage(title: 'My Awesome App'),
+    );
+  }
+}
+{% endprettify %}
+
+Probablemente ya conozcas la aplicación de ejemplo de Flutter Gallery.
+Para utilizar la ventana de rendimiento con Flutter Gallery, usa la copia en el directorio de 
+[ejemplos](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery)
+que fue instalado con Flutter, y ejecuta la aplicación en modo profile. El programa se escribe de manera que el menú de la app le permita cambiar dinámicamente la ventana superpuesta,
+así como habilitar la comprobación de llamadas a `saveLayer` y la presencia de imágenes en caché.
+
+<aside class="alert alert-info" markdown="1">
+**Nota:** No puede habilitar la ventana de rendimiento en la app Flutter Gallery descargada desde el App Store. Esa versión de la aplicación está compilada en modo release (no en modo profile), y no proporciona un menú para habilitar o deshabilitar la ventana superpuesta.
+</aside>
+
+### Identificando problemas en la UI de gráficas
+
+Si la ventana de rendimiento aparece en rojo en el gráfico de interfaz de usuario, comienza por perfilar la VM de Dart, incluso si el gráfico de la GPU también aparece en rojo. Para ello, utiliza la herramienta de profile de Dart
+[Observatory](https://dart-lang.github.io/observatory/).
+
+#### Mostrando el Observatory
```

Comment:
la herramienta Observatory

Reply:
To do this, use Observatory, Dart’s profiling tool. Esta sentencia traduciría:
Para ello, utiliza 
[Observatory](https://dart-lang.github.io/observatory/), la herramienta para profile de Dart.